install/linux/usr/bin/odemis-mic-selector

```diff
@@ -64,43 +63,70 @@ def count_usb_device(usbid):
     return out.count(usbid)
 
 
+def find_backend_pcus() -> List[model.HwComponent]:
+    """
+    return (list of HwComponent): all the power control units found
+    :raises: IOError if no power control unit is found
+    """
+    pcus = []
+    try:
+        comps = model.getComponents()
+    except (Pyro4.errors.CommunicationError, IOError, LookupError):
+        raise IOError("Failed to access the backend")
+    for comp in comps:
+        if comp.role and comp.role.startswith("power-control"):
+            pcus.append(comp)
+    return pcus
+
+
+def connect_all_pcus() -> List[model.HwComponent]:
+    pcus = []
+    ports = glob.glob(PCU_PORT_PATTERN)
+    # For testing: uncomment this line:
+    # ports.append("/dev/fake")
+    for port in ports:
+        try:
+            pcu = powerctrl.PowerControlUnit(name=f"power control unit on {port}", role=None, port=port, check_power=False)
+            pcus.append(pcu)
+        except Exception as ex:
+            # Not all device on the port are PCU, so we can ignore the error
+            logging.warning("Failed to connect to PCU on %s: %s", port, ex)
+            continue
+
+    return pcus
+
+
 def get_eeprom_ids():
     """
     return (set of int): ID of each EEPROM detected
     """
     terminate = False
     try:
-        pcu = model.getComponent(role="power-control")
+        pcus = find_backend_pcus()
         logging.debug("Using the backend to access the power controller")
-    except (Pyro4.errors.CommunicationError, IOError, LookupError):
-        logging.debug("Failed to access the backend, will try directly")
-        pcu = CLASS_PCU(**KWARGS_PCU)
+    except IOError:
+        pcus = connect_all_pcus()
```

Comment:
This part is quite confusing, if the backend cannot be accessed and if no PCU is found, try to connect all pcu's?
Maybe add some more comments to clarify this way of connecting?